pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -203,40 +202,10 @@ func getPipelineRunUpdates(t *testing.T, actions []ktesting.Action) []*v1beta1.P
 	return nil
 }
 
-func TestReconcile(t *testing.T) {
-	testCases := []struct {
-		name              string
-		embeddedStatusVal string
-	}{
-		{
-			name:              "default embedded status",
-			embeddedStatusVal: config.DefaultEmbeddedStatus,
-		},
-		{
-			name:              "full embedded status",
-			embeddedStatusVal: config.FullEmbeddedStatus,
-		},
-		{
-			name:              "both embedded status",
-			embeddedStatusVal: config.BothEmbeddedStatus,
-		},
-		{
-			name:              "minimal embedded status",
-			embeddedStatusVal: config.MinimalEmbeddedStatus,
-		},
-	}
-
-	for _, tc := range testCases {
-		t.Run(tc.name, func(t *testing.T) {
-			runTestReconcileWithEmbeddedStatus(t, tc.embeddedStatusVal)
-		})
-	}
-}
-
-// runTestReconcileWithEmbeddedStatus runs "Reconcile" on a PipelineRun with one
+// runTestReconcile runs "Reconcile" on a PipelineRun with one
 // Task that has not been started yet.  It verifies that the TaskRun is created,
 // it checks the resulting API actions, status and events.
-func runTestReconcileWithEmbeddedStatus(t *testing.T, embeddedStatus string) {
+func TestReconcile(t *testing.T) {
 	t.Helper()
```

Comment:
t.Helper should be removed, here and in all the other test classes that are no longer helpers.